server/security.py

```
@@ -0,0 +1,210 @@
+    if "post_down" in out:
+        out["post_down"] = validate_wireguard_hook(str(out.get("post_down", "")), "post_down")
+    return out
+
+
+def validate_traffic_bytes(value: float, field_name: str = "bytes") -> float:
+    if value is None:
+        return 0.0
+    try:
+        num = float(value)
+    except (TypeError, ValueError):
+        raise HTTPException(status_code=400, detail=f"Invalid {field_name}")
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)